web-ui/src/components/volunteer/VolunteerTables.jsx

```diff
@@ -0,0 +1,80 @@
+import PropTypes from 'prop-types';
```

Comment:
This component provides a tabbed interface with three tabs for these components:
Organization, VolunteerRelationships, and VolunteerEvents.